src/functions-reference/covariance_matrix_distributions.Rmd

```diff
@@ -110,6 +111,52 @@ lower-triangular Cholesky factor of the scale matrix `L_S`; may only be used in
 transformed data and generated quantities blocks
 `r since("2.30")`
 
+## Inverse Wishart distribution, Cholesky Parameterization (#inverse-wishart-cholesky-distribution)
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in (K-1, \infty)$, and $L_S \in
+\mathbb{R}^{K \times K}$ is lower triangular and such that $L_S L_S^{\top}$
+is positive definite,then for
+$L_W \in \mathbb{R}^{K \times K}$ is lower triangular and such that $L_WL_W^{\top}$
+is positive definite, \[
+\text{InvWishartCholesky}(L_W \mid \nu,L_S) =
```

Comment:
I think we should put the wishart and inverse wishart before the Cholesky versions.  As is, the sections are 

* Wishart
* Wishart Cholesky
* inv Wishart Cholesky
* inv Wishart

But I think inv Wishart should come before inv Wishart Cholesky.  

Inv wishart is presented as

inv_wishart(W | nu, S)

so I think 

inv_wishart_cholesky(L_W | nu, L_S)

is the notation we should use, with subscripts on the L that match the Cholesky parameters.

Then we want to say that inv_wishart_cholesky(L_W | nu, L_S) is the density of a random variable defined as the Cholesky factorization of a dry from wishart_cholesky(nu, L_S * L_S').
```
W ~ wishart(nu, Sigma)
```

so I think for inverse Wishart with Cholesky

inv_wishart(Y | W, nu)